unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/Tests/BIWGizmosShould.cs

```diff
@@ -62,31 +65,37 @@ public void TestTranslateGizmosType()
     {
         //Arrange
         gizmosController.SetGizmoType(BIWSettings.TRANSLATE_GIZMO_NAME);
-        DCLBuilderGizmo translateGizmo = gizmosController.activeGizmo;
+        IBIWGizmos gizmo = gizmosController.activeGizmo;
 
         //Assert
-        Assert.IsInstanceOf(typeof(DCLBuilderTranslateGizmo), translateGizmo);
+        Assert.IsInstanceOf(typeof(IBIWGizmos), gizmo);
     }
 
     [Test]
     public void TestRotateGizmosType()
     {
         //Arrange
         gizmosController.SetGizmoType(BIWSettings.ROTATE_GIZMO_NAME);
-        DCLBuilderGizmo gizmo = gizmosController.activeGizmo;
+        IBIWGizmos gizmo = gizmosController.activeGizmo;
 
         //Assert
-        Assert.IsInstanceOf(typeof(DCLBuilderRotateGizmo), gizmo);
+        Assert.IsInstanceOf(typeof(IBIWGizmos), gizmo);
```

Comment:
We should be asserting against `DCLBuilderRotateGizmo` here right?

Reply:
No, we have decoupled Builder and BuilderInWorld and this test is from BuilderInWorld so it should assert against the BIW type